scripts/k8s.env

```diff
@@ -72,14 +72,16 @@ ETCD_VERSION=3.5.16
 #   template creation process.
 
 ### Debian 12 Image (Bookworm)
+TEMPLATE_VM_BRIDGE=vmbr0
```

Comment:
Could you place this up near line 12?

Reply:
Oh, and could you add the `TEMPLATE_VM_BRIDGE` variable somewhere near line 280 in `clustercreator.sh`? To ensure that the script does not run if the user did not define it.

Reply:
Done